test/forms/TelephoneNumberFormProviderSpec.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.behaviours.StringFieldBehaviours
+import play.api.data.FormError
+import wolfendale.scalacheck.regexp.RegexpGen
+
+class TelephoneNumberFormProviderSpec extends StringFieldBehaviours {
+
+  val requiredKey = "telephoneNumber.error.required"
```

Comment:
We've referred to every Class/Object/Instance for this feature as `TelephoneNumber{Form/Page/Controller}` where it should be `TrusteeTelephoneNumber...`

Reply:
Refactor once merged